template/_package.json

```diff
@@ -72,7 +72,7 @@
     "eslint-plugin-prettier": "^3.0.1",
     "prettier": "^1.16.4"<% } %><% if (ui === 'tailwind') { %>,
     "autoprefixer": "^8.6.4",
-    "tailwindcss": "^0.7.0"<% } else if (ui === 'vuetify') { %>,
+    "tailwindcss": "^1.0.0"<% } else if (ui === 'vuetify') { %>,
```

Comment:
1.0.0 had issues, could you make it `1.0.1`?

Reply:
`^1.0.0` will automatically pick the latest minor version sir

Reply:
@bart yes it'll, but why not keep the base package latest? So, if `1.0.2` is released tomorrow, ideally it should be `1.0.2` as base.

Reply:
I don't see any any advantage here. Result will be the exact same.

Reply:
It will be cover all of 1.x version of tailwindcss. 
Thank you for your time for review @vinayakkulkarni :)